internal/db/schema/migrations/oss/postgres/29/04_kms_keys.up.sql

```diff
@@ -0,0 +1,215 @@
+begin;
+
+-- make the required schema changes to adopt
+-- github.com/hashicorp/go-kms-wrapping/extras/kms/v2 
+-- this migration is from 04_keys.up.sql in that repo. 
+
+
+-- we can use our existing kms_root_key table.
```

Comment:
I am wondering if we want to use the existing table, or rename this and create a new table. We rename some other tables with `deprecated`, could there be a reason to do the same thing here? Or is there a reason that we definitely should use the existing table?

Reply:
I think, there's no reason to re-create and migrate these tables. What value is there here to doing this?

Reply:
Yea, I am not sure there is a lot of value, it might just be a consistency things where all of the old tables are renamed as deprecated and otherwise preserved, I don't feel too strongly about it, just thinking out loud since this was one noticeable part of the diff between this file and the one in kms.

Reply:
At this point, I think it's best to not recreate these tables.